### DISCUSSION TITLE
renaming useId composable

### DISCUSSION THREAD

**Discussion by hubdidub** [quasarframework/quasar#17497] [Ideas / Proposals] [answered]
After Quasar introduced the new `useId` composable, Vue has now also added a similar one with the same name ([link to docs](https://vuejs.org/api/composition-api-helpers#useid)).  
Not that there is a real problem with that, but it could be a little bit confusing, don't you think?

Well, I guess it's not possible anyway since it's already been released, but wouldn't it be a good idea of renaming it, maybe to `useUID` to differentiate it from Vue's variant? It's different alter all because it doesn't return just a number.

**Comment by rstoenescu** [comment, marked as answer]
Well, this is an unfortunate event. But we can't simply rename our composable as this would be a breaking change.
However, posting it here for others to see how they can import both in the same file:

```js
import { useId as qUseId } from 'quasar'
import { useId as vueUseId } from 'vue'
```
